### ISSUE TITLE
multiple select boxes in single page

### ISSUE THREAD

**Issue by mahiagit** [Dogfalo/materialize#2099]
We are using multiple Select Boxes and in all the select boxes when we select like "add " crates new option for the select box and made it to be shown as selected. it works fine in all cases as we reinitialize the stuff as $("#something").material_select();

The problem appears when we create items for the select boxes the the latest select boxes give the right index i.e the selected one. others (the one form where we create the option) show the first index and not the one which is being shown as selected.

Basically When we reinitialize the select box with the id the older used select box shows the selected one but done holds the selected value.

**Comment by acburst** [comment]
Can you post a codepen?
